client/src/constants/data-types.js

```diff
@@ -2,3 +2,9 @@ export const linkTypes = {
   LINK: 'LINK',
   PHONE: 'PHONE',
 };
+
+export const contactLinksTypes = {
+  PHONE: 'PHONE',
+  WEBCHAT_LINK: 'WEBCHAT_LINK',
+  EMAIL: 'EMAIL',
+};
```

Comment:
this is same as `linkTypes` above, i think you should add webchat link

Reply:
done